### PR TITLE
Alex/intake

### PR DESCRIPTION
**Pull request by AlexCook75921312312312**
I think that because of clean up the code is now incompatible with the drivetrain stuff

### CODE REVIEW

File: src/org/usfirst/frc/team4915/stronghold/commands/ExtendLauncherCylinderCommand.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import edu.wpi.first.wpilibj.command.Command;
+import org.usfirst.frc.team4915.stronghold.Robot;
+
+public class ExtendLauncherCylinderCommand extends Command {
+
+    public ExtendLauncherCylinderCommand() {
+        requires(Robot.intakeLauncher);
+    }
+
+    protected void initialize() {
```
Comment (phroa) [inline src/org/usfirst/frc/team4915/stronghold/commands/ExtendLauncherCylinderCommand.java]: consider running `Code -> Clean Up...` on your src folder to automatically add the `@Override` annotations that are missing from these methods.

File: src/org/usfirst/frc/team4915/stronghold/subsystems/DriveTrain.java [hunk 2]
```diff
@@ -36,8 +34,8 @@ public double modifyThrottle() {
 
     private void setMaxOutput(double topSpeed) {
         // TODO Auto-generated method stub
```
Comment (phroa) [inline src/org/usfirst/frc/team4915/stronghold/subsystems/DriveTrain.java:36]: I think you left a `TODO` here

**Comment by AlexCook75921312312312** [comment]
When I used clean code to put the override statements back it seems to have had the side effect of filling my code with "this.". I will fix that during the next pull

**Comment by phroa** [comment]
that's actually intentional

**Comment by phroa** [comment]
oh, it looks like you committed the `FRCUserProgram.jar` in the `dist` folder. that should probably go.

**Comment by AlexCook75921312312312** [comment]
I have no memory of changing that

**Pull request merged**